### DISCUSSION TITLE
Format of some files inside btc_mainnet\db folder

### DISCUSSION THREAD

**Discussion by Android-X13** [bisq-network/bisq#6277] [Q&A]
What is the format of files like `ClosedTrades`? A lot of info is in the clear but other parts of these files cannot be read. 
Is there a way to decode them outside of Bisq application?

**Comment by ghost** [comment]
https://github.com/bisq-network/bisq/issues/6278#issuecomment-1176634346

**Reply by Android-X13** [reply]
Sometimes I am able to decode PreferencesPayload with Google's protoc binary, other times it fails to parse the input.
It always fails to parse ClosedTrades for some reason

**Reply by chimp1984** [reply]
I assume the protoc tool cannot parse data which contains custom data types. Primitive values will likely work. PreferencesPayload has likely mostly primitive data types.
